docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+                    "showDesc" : boolean
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+
+`options.order`: Required array of strings. An array of the names of `fields` or `helpers`, in order, that are to be
+                 formatted within this group. It may contain any field, but only helpers of the type "static".
+
+`options.showDesc`: Boolean. Default is true. True or false that the section's `description` is displayed as the header for
+                    section of elements.
+
+An implementation could choose to set this key to false, and display an independent "static" helper as a header instead.
+ 
+# 5.0: UI Properties
+
+The UI component of the Z-JSON Instrument contains information on how we render our `fields` and `helpers` objects
+on the front-end. It is composed of JSON objects whose keys match those of the field or helper object it's describing.
+This component should only make decisions on which HTML or HTML Form element those objects get rendered as,
+and nothing else. It may also provide additional UI properties that assist in the rendering.
+
+The UI component object has the following general format:
+
+```js
+{
+    ...
+    "ui" : {
+        "nameOfFieldOrHelper" : {
+            "type": string,
+            "options": {
+                ...
+            },
+        },
+        ...
+    }
+}
+```
+
+A UI object type may or may not have type-specific `options` that configure the rendering of the front-end element.
+
+## 5.1 UI Types
+
+For fields:
+
+- [checkbox](#511-checkbox-element)
+- [date](#512-date-element)
+- [number](#513-numeric-element)
+- [radio](#514-radio-button-element)
+- [select](#515-select-element)
+- [text](#516-text-element)
+- [textarea](#517-textarea-element)
+- [time](#518-time-element)
+
+For helpers:
+
+- [header](#519-header-element) (from h1 to h6)
+- [link](#5110-link-element)
+- [static](#5111-static-element)
+
+These are all mappable to some LORIS React Form element, except for "header" which is native to HTML.
+
+### 5.1.1 Checkbox Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "checkbox"
+        }
+    }
+}
+```
+
+### 5.1.2 Date Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "date"
+        }
+    }
+}
+```
+
+### 5.1.3 Numeric Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "number"
+        }
+    }
+}
+```
+
+### 5.1.4 Radio Button Element
+
+```js
+{
+    "ui": {
+        ...
+        "nameOfRadioButtons" : {
+            "type" : "radio",
+            "options" : {
+                "vertical" : boolean
+            }
+        }
+    }
+}
+```
+
+`options.vertical`: Boolean. True or false that the radio group is rendered vertically. The default value is false, or
+horizontally. 
+
+### 5.1.5 Select Element
+
+```js
+{
+    "ui": {
+        "nameOfField" : {
+            "type" : "select"
+        }
+    }
+}
+```
+
```

Comment:
I think all of the above need descriptions of what they mean.. I'd also call "select"s "dropdown"s, because that's less specific to HTML.

Reply:
i think we should keep 'select' because there's also multiselect elements which doesn't make sense as 'multidropdown'